include/deal.II/grid/tria_description.h

```diff
@@ -446,6 +448,24 @@ namespace TriangulationDescription
         TriangulationDescription::Settings::default_setting,
       const unsigned int my_rank_in = numbers::invalid_unsigned_int);
 
+    /**
+     * Similar to the above function but the owner of active cells are provided
+     * by a cell vector (see also
+     * parallel::TriangulationBase::global_active_cell_index_partitioner() and
+     * CellAccessor::global_active_cell_index()). This function allows to
+     * repartition distributed Triangulation objects.
+     *
+     * @note The communicator is extracted from the Triangulation @p tria.
+     *
+     * @note The multgrid level is currently not constructed, since @p partition
+     *   only describes the partitioning of the active level.
```

Comment:
> I guess you want, at some point, maybe add the Settings argument?

Not for now; at the moment, I only need the active level. Extending the concept to the other levels is a bit more involved since one would need to think about how to partition these levels (first-child rule?) and if this is clear one would need to introduce ghost cells on those levels too which is not done at the moment.

Reply:
I totally agree that level dofs are not really the goal for the global coarsening algorithms and I don't see a pressing need here; I was primarily commenting on the interface analogy with the rest of the class.